Apply page

phi-h-nguyen opened the pull request:
Finished apply page (optimized for mobile, but not for tablets/anything between desktop and a phone), and normalized nav links on all pages for desktop and mobile. Sabrina will go in later to change some of the styling and content/links for the apply page

mikinty commented on pages/apply.html:
```diff
@@ -11,52 +11,101 @@
+    integrity="sha384-OgVRvuATP1z7JjHLkuOU7Xw704+h835Lr+6QL9UvYjZE3Ipu6Tp75j7Bh/kR0JKI"
+    crossorigin="anonymous"></script>
+
+  <style>
```
Can you create a new css file for these styles?

sabrinazhai replied:
Those are the bootstrap styles right? Do you mean saving a copy of those bootstrap files in our local folder instead of calling them in this line?

mikinty replied:
No specifically these custom inline css in the html

mikinty replied:
these are fixed now, moved to `apply.css`

mikinty reviewed (changes requested):
Hey overall looks good!

From the figma for desktop they wanted to make it one page, so if you could tighten up the margins so it fits on one page that would be good

![image](https://user-images.githubusercontent.com/12106484/92314259-77fd0600-efa3-11ea-9e68-a9deb4d32281.png)

For mobile, they wanted to make it one entry per row, and a long scrollable, but right now I'm getting multiple entries per row.

![image](https://user-images.githubusercontent.com/12106484/92314298-c7dbcd00-efa3-11ea-9012-3a314d715a5a.png)

I don't think the DIN fonts are loading? Also really nitpicky, but the designers wanted the @ to be helvetica and everything else to DIN.

![image](https://user-images.githubusercontent.com/12106484/92314282-abd82b80-efa3-11ea-842a-c6cea8fbe493.png)

mikinty commented:
For the font, what I would do is make a special class called like font-alt or font-helvetica or something and do like 

```
Stuff <span class="font-alt">@<span/> UC Conan 
```

phi-h-nguyen commented:
updates: fixed apply page to fit on one page, cols are uniform and collapse on mobile, normalized nav links and added an underline for the active page, added correct fonts, fixed up css, added a text highlight color, added favicons

mikinty commented:
I made the headers bold for web
Migrated some css so it's easier to manage (inline html css is generally frowned upon)

![image](https://user-images.githubusercontent.com/12106484/92318147-05594e00-efd6-11ea-968d-b8bce0e700cf.png)

mikinty commented on pages/apply.html:
```diff
@@ -11,52 +11,125 @@
+  <script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js"
+    integrity="sha384-Q6E9RHvbIyZFJoft+2mJbHaEWldlvI9IOYy5n3zV9zzTtmI3UksdQRVvoxMfooAo"
+    crossorigin="anonymous"></script>
+  <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/js/bootstrap.min.js"
```
And we also don't clutter our repo by committing a 10k line file (even tho our repo is kinda already riddled by these vendors already)

mikinty commented:
I removed the vendor, going to do a history to remove the bootstrap

mikinty commented:
Rebased, removing the bootstrap css local version and using CDN

mikinty commented:
Added mobile styles, see slack

mikinty commented:
Only issue is because of the way the original background fade-in was implemented (6s transition duration...wtf), if you change the dimensions of the website on browser, you'll see some real sketchy stuff. I think for now that's an edge use case so we can fix that in the future...

mikinty commented:
Ok for future PR:

- Scroll background image on apply (and in general), probably need to abandon the `position: absolute` css for the background
- I'm not actually sure if want a fixed header for mobile (or even for web), see 
![image](https://user-images.githubusercontent.com/12106484/92331524-463c7b80-f045-11ea-9d7b-2f2da436363b.png)

mikinty commented:
But overall I think this is good to go 👍
